### PR TITLE
[hailctl] Fix cluster creation

### PR DESCRIPTION
**Pull request by tpoterba**
Python is terrible.

### CODE REVIEW

File: hail/python/hailtop/hailctl/dataproc/resources/init_notebook.py [hunk 1]
```diff
@@ -40,15 +40,13 @@ def mkdir_if_not_exists(path):
...
-        'jupyter_console<5',
         'nbconvert<6',
         'notebook<6',
         'qtconsole<5',
-        'jupyter', 'tornado<6',  # https://github.com/hail-is/hail/issues/5505
         'lxml<5',
         'google-cloud==0.32.0',
-        'ipython<7',
         'jgscm<0.2',
+        'jupyter', 'tornado<6',  # https://github.com/hail-is/hail/issues/5505
```
Comment (akotlar) [inline hail/python/hailtop/hailctl/dataproc/resources/init_notebook.py]: lets pin the jupyter version

Comment (tpoterba) [reply hail/python/hailtop/hailctl/dataproc/resources/init_notebook.py]: we can't :/ Jupyter basically uses shim versions

Comment (akotlar) [reply hail/python/hailtop/hailctl/dataproc/resources/init_notebook.py]: I can accept, not sure if you want dan to look

**Comment by tpoterba** [comment]
🤷‍♂  he saw at office hours

**Comment by akotlar** [comment]
He's busy, I'll accept

**Pull request merged**